classes/module/Module.php

```diff
@@ -879,6 +879,10 @@ public function enable($force_all = false)
             }
         }
 
+        // set active to 1 in the module table
+        $sqlEnable = 'UPDATE %smodule SET active = 1 WHERE id_module = %s';
+        Db::getInstance()->execute(sprintf($sqlEnable, _DB_PREFIX_, (int) $this->id));
```

Comment:
```suggestion
        Db::getInstance()->update('module', ['active' => 1], 'id_module = '. (int) $this->id);
```
With the remove of `$sqlEnable` var.